modules/core/src/test/java/org/apache/ignite/internal/processors/cache/transactions/AtomicOperationsInTxTest.java

```diff
@@ -58,122 +61,186 @@ public class AtomicOperationsInTxTest extends GridCommonAbstractTest {
     /** {@inheritDoc} */
     @Override protected void beforeTestsStarted() throws Exception {
         super.beforeTestsStarted();
+    }
 
+    /** {@inheritDoc} */
+    @Override protected void beforeTest() throws Exception {
```

Comment:
Let's revert changes in beforeTest and afterTest, and start grid in `beforeTestsStarted` as it was before.

Reply:
Thank you, I will do it

Reply:
As was agreed later, beforeTest and afterTest are left.
beforeTestsStarted is removed